pkg/agent/datamodel/sig_config.go

```diff
@@ -495,55 +526,111 @@ var (
 		Version:       LinuxSIGImageVersion,
 	}
 
-	SIGCBLMarinerV2ImageConfigTemplate = SigImageConfigTemplate{
+	SIGAzureLinuxV2Gen1ImageConfigTemplate = SigImageConfigTemplate{
+		ResourceGroup: AKSAzureLinuxResourceGroup,
+		Gallery:       AKSAzureLinuxGalleryName,
+		Definition:    "V2",
+		Version:       LinuxSIGImageVersion,
```

Comment:
this needs to change to 202309.11.0, once this image is tested e2e through RP, then we can sync with regular sig images.

Reply:
Fixed